### ISSUE TITLE
2016 consumed media

### ISSUE THREAD

**Issue by rick** [rick/life#8]
![](http://www.bforbel.com/wp-content/uploads/2013/05/books-and-movies.jpg)

**Comment by rick** [comment]
**January**
- [ ] :book: The Martian
- [ ] :book: The Road
  - zomg, depressing
- [ ] :book: Ready Player One
- [ ] :tv: Rick & Morty, Season 1
- [ ] :book: parts of "Continuous Delivery"
- [ ] :book: half of "Test Driven Development for Embedded C"
- [ ] :tv: Sherlock: The Abominable Bride

**Comment by rick** [comment]
**February**
- [ ] :book: American Gods
- [ ] :tv: Rick & Morty, Season 2
- [ ] :book: I am Legend
- [ ] :book: The Magicians
- [ ] :movie_camera: Dead Pool
- [ ] :book: Hyperion
- [ ] :movie_camera: (most of) Everest
- [ ] :book: Fall of Hyperion

**Comment by rick** [comment]
**March**
- [ ] :movie_camera:  [Control](http://www.imdb.com/title/tt0421082/)
- [ ] :book: Foundation

**Comment by rick** [comment]
Recommendations: 
- [ ] "You can check out the song Daaru Peeke Dance (From Kuch Kuch Locha Hai) by Neha Kakkar, Aishwarya Nigam & Amjad Nadeem at http://www.soundhound.com/?t=100254897561855614"
- [ ] :movie_camera: [Coherence](http://www.imdb.com/title/tt2866360/)

**Comment by rick** [comment]
March / April / May
- 🎥  Mockingjay: hunger games part 2
- :book: The Magician King
- :book: The Magician's Land
- 📺  DareDevil, Season 2
- 🎥  Ex Machina
- 🎥  Man Up
- :tv: Silicon Valley, Seaons 3 (early episodes)
- :tv: Mr. Robot, Season 1 (episodes 1-5)
- 🎥  Hateful Eight
- 🎥  Spectre
- 🎥  Straight Outta Compton
- 🎥  Captain America: Civil War

**Comment by rick** [comment]
Europe travel notes (things worth jotting down):
- 🎵  techno playlist:  https://soundcloud.com/rodhad/radio-one-essential-mix-rodhad
- 🎵  Ben L'Oncle Soul (spec. "Seven Nation Army" cover)
- 🎵  "Way Back When" by Kodaline
- 🎵  "Take a step" by Milo Greene
- 🎵  "the stripes" (band)
- 🎵  Ben Kaplan
- 🎵  benjamin clementine
- 🎵  Ben Kaplan
- 🎨  Jun'ya ishigama (artist)

recommended to me:
- 🎵  "The Colour In Anything" (album) by James Blake
- 🎵  [Rødhåd - Radio One Essential Mix by Rødhåd](https://soundcloud.com/rodhad/radio-one-essential-mix-rodhad)
- 🎵  [Live at Factory, Osaka - 26.09.15 by Ben UFO](https://soundcloud.com/benufo/live-at-factory-osaka-260915)
- 🎵  [AGORIA quailty tech DJ set: Mixmag & W Amsterdam Sessions](https://youtu.be/194ZcUDYDl0)
- 🎵  [Minimal Techno](https://open.spotify.com/user/alekzanther/playlist/0B3WoheGNqol1B69LM9Y8n)
- 🎵  [Berghain Music](https://open.spotify.com/user/1110774884/playlist/2hqbSYinuSlYwgQBOX81Yo)
- 🎵  [Les La Bas (Bonobo Remix)](https://play.spotify.com/album/71zbNaktwQKrJ87FxYLnPb?play=true)
- 🎵  [Circles EP](https://open.spotify.com/album/2YblKj7ixayK5qOGZujayy) by Bicep

**Comment by rick** [comment]
Art stuff seen, worth following up on:
- Chris Leabeau
- Bernadette Corporation
- Yutaka Takanashi (cool architectural stuff)
- John Heartfield and AIZ
- Kasimir Malevitch

**Comment by rick** [comment]
Cc @Vinbarnes you might also enjoy Ben Caplan [video](https://youtu.be/kfzy08FSdS4)

**Comment by rick** [comment]
recommended: 
- [ ] :book: [Iron, Steam, and Money](https://www.amazon.com/Iron-Steam-Money-Industrial-Revolution/dp/184595212X/188-5339634-0176628?ie=UTF8&*Version*=1&*entries*=0)

**Comment by rick** [comment]
More recommendations:
- [ ] 🎹  soundtrack to "the last of us"
- [ ] :book: mushrooms demystified
- [ ] :book: buddha on the midnight sea

**Comment by jhosman** [comment]
> 📖  mushrooms demystified

Ooh, that looks interesting. This reminds me of a great book I read a while back: [Magical Mushrooms, Mischievous Molds](https://www.amazon.com/Magical-Mushrooms-Mischievous-George-Hudler/dp/0691070164)

**Comment by rick** [comment]
Word -- and now I don't even have to add it to my list! :bow:

I have it on good authority that [Paul Stamets](http://www.amazon.com/Paul-Stamets/e/B001JS0KV6) writes a ton of good mushroom books, largely on the "grow your own" side of the topic.

**Comment by jhosman** [comment]
I have a few of his books. They're great to use as growing resources -- Pretty hefty to read cover to cover 😄  An ecologist friend of mine who works with fungus grumbles about Stamets all the time. Something about him being a fungus sensationalist, ha. I still think he's fascinating though! Love his [TED Talk](https://www.ted.com/talks/paul_stamets_on_6_ways_mushrooms_can_save_the_world?language=en).

**Comment by rick** [comment]
Yes! I love that talk too. Still haven't grabbed any of his books as of yet, though. One day.

**Comment by rick** [comment]
Hiatus here.  I forgot a bunch of 💩 I consumed.  Such it is.
- [x] 🎵 [Thao & The Get Down Stay Down](http://www.thaoandthegetdownstaydown.com/)
- [ ] 🎵  (in progress) [Laura Gibson](http://www.lauragibsonmusic.com/)
- [x] :book: [The Production of Houses](https://www.amazon.com/Production-Houses-Center-Environmental-Structure/dp/0195032233) by Christopher Alexander
  
  Many of y'all know I have a total crush on Christopher Alexander and basically almost everything he's ever written.  This book was on my radar for like 10 years but I never got a copy until recently. Timely, as I'm doing extensive remodeling on a standalone 2-story building on our property. Dig it.
- [ ] :book: (in progress) [The Tibetan Yogas of Dream and Sleep](https://www.amazon.com/Tibetan-Yogas-Dream-Sleep/dp/1559391014/ref=sr_1_1?s=books&ie=UTF8&qid=1475105721&sr=1-1&keywords=tibetan+yogas+of+dream+and+sleep) by Tenzin Wangyal Rinpoche  -- Good shit.
- [ ] :book: (in progress) [How to Meditate](https://www.amazon.com/How-Meditate-Practical-Making-Friends/dp/1604079339/ref=sr_1_1?s=books&ie=UTF8&qid=1475105747&sr=1-1&keywords=how+to+meditate) by Pema Chodron -- Replacing my standard "intro to meditation book" (Mindfulness in Plain English) with this going forward. Draws on Lojong for some of the direction.
- [x] 🎥 [Jason Bourne](http://www.imdb.com/title/tt4196776/) I love me some Jason Bourne movies. I might not watch this one 40,000 times like the first 3, but I still enjoyed it.

**Comment by rick** [comment]
wishlist (shit I want to get):
- [ ] [The Profound Treasury of the Ocean Dharma](http://www.shambhala.com/profoundtreasury.html) (also available for more :dollar: on amazon; and occasionally used versions show up at powell's)

**Comment by rick** [comment]
wishlist: Linz Cafe by Christopher Alexander

**Comment by rick** [comment]
to watch: https://www.youtube.com/watch?v=PAC7Anmr6X8

**Comment by rick** [comment]
follow up on:

 - [ ]  paradinas/u-Ziq  (see also if there's a video/stream of recent club show in Shoreditch)
